.github/workflows/e2e.yml

```diff
@@ -1,33 +1,43 @@
 name: E2E tests
 on:
+  workflow_call:
+  workflow_dispatch:
   push:
-    branches: [ main ]
+    branches: [ main, feat/workflow-for-e2e ]
```

Comment:
**Question:** as I understand it, we only want to run these tests after a deploy? This already happens on push to the main branch, so probably we can remove the `push` directive here.